src/test/java/io/kokuwa/maven/helm/HelmMojoTest.java

```diff
@@ -37,8 +38,9 @@ class Executable {
 		void fixed(LintMojo mojo) {
 			mojo.setUseLocalHelmBinary(true);
 			mojo.setAutoDetectLocalHelmBinary(false);
-			mojo.setHelmExecutableDirectory(new File("src/it"));
-			Path expected = Paths.get("src/it").resolve(HELM).toAbsolutePath();
+			mojo.setHelmExecutableDirectory(MojoExtension.determineHelmExecutableDirectory());
+			Path expected = Paths.get(MojoExtension.determineHelmExecutableDirectory().toString())
+					.resolve(HELM).toAbsolutePath();
```

Comment:
this was added because when debugging, the included helm executable would not run on aarch64 Mac machines, so now we have a function to check the machine architecture and use the correct helm executable (also added an aarch64 helm executable)